sensor/common/centralclient/client.go

```diff
@@ -84,8 +85,11 @@ func NewClient(endpoint string) (*Client, error) {
 		},
 	}
 	httpClient := &http.Client{
-		Transport: &http.Transport{TLSClientConfig: tlsConf},
-		Timeout:   requestTimeout,
+		Transport: &http.Transport{
```

Comment:
Is there a wrapper HTTP client to avoid this configuration error in future implementations?
We could also add a roxvet check to avoid using `http.Client` directly, instead using StackRox's wrapper.

Reply:
FWIW, we have `proxy.UseWithDefaultTransport()` which IIUC can be applied to all transport instances in the module.

Reply:
That's already invoked within [the main init](https://github.com/stackrox/stackrox/blob/master/central/main.go#L222) to add the proxy to the `http.DefaultTransport`. So, in case anyone reuses the `http.DefaultTransport`, it will have our proxy configuration. It won't help in this particular problem though, since clients may still define their own transport instead of reusing the default one (e.g. because of specific TLS config requirements).

Reply:
> So, I suppose we have to keep it as-is, I'll try to check how easy it is to add a roxvet check ensuring that iff you use a http.Client, either proxy.RoundTripper() or proxy.FromConfig() is given, but I'll do that in a separate PR.

This sounds good, thanks for investigating!